bootstrap/http/src/main/kotlin/com/studentcenter/weave/bootstrap/user/api/UserApi.kt

```diff
@@ -69,4 +70,13 @@ interface UserApi {
         request: UserSetMyHeightRequest
     )
 
+    @Secured
+    @Operation(summary = "Set My Animal Type")
+    @PatchMapping("/my/animal-type")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    fun setMyAnimalLike(
```

Comment:
p4) Like라는 의미가 크게 중요하진 않은 것 같아요!
```suggestion
    fun setMyAnimalType(
```

Reply:
에공 이거 안바꿨네용 👍